backend/endpoints/list_version.go

```diff
@@ -0,0 +1,24 @@
+package endpoints
+
+import (
+	"context"
+
+	"github.com/dwarvesf/smithy/backend/config"
+	"github.com/dwarvesf/smithy/backend/service"
+	"github.com/go-kit/kit/endpoint"
+)
+
+type listVersionResponse struct {
```

Comment:
ListVersion endpoint should have more meanful feature like version sequence name, timestamp of last sync..etc